### PR TITLE
Animation importing

### PR DESCRIPTION
**Pull request by ousttrue**
Implement Animation Importing.
And, add sample scene from https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/AnimatedCube

### CODE REVIEW

**Comment by stevenvergenz** [comment]
@blgrossMS I've updated this to match the rest of the codebase. Take a look?

File: GLTFSerialization/GLTFSerialization/GLTFHelpers.cs [hunk 1]
```diff
@@ -584,9 +616,7 @@ private static void MergeAnimationsAndSkins(GLTFRoot mergeToRoot, GLTFRoot merge
 
 					foreach (AnimationChannel channel in animation.Channels)
 					{
-						SamplerId samplerId = channel.Sampler;
-						samplerId.Id += previousGLTFSizes.PreviousSamplerCount;
-						samplerId.Root = mergeToRoot;
+						// NOTE: don't need to repoint sampler references
```
Comment (blgrossMS) [inline GLTFSerialization/GLTFSerialization/GLTFHelpers.cs]: this does need to exist still

File: GLTFSerialization/GLTFSerialization/Schema/AnimationChannel.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@ public class AnimationChannel : GLTFProperty
 		/// The index of a sampler in this animation used to compute the value for the
 		/// target, e.g., a node's translation, rotation, or scale (TRS).
 		/// </summary>
-		public SamplerId Sampler;
+		public int Sampler;
```
Comment (blgrossMS) [inline GLTFSerialization/GLTFSerialization/Schema/AnimationChannel.cs]: This should still be a SamplerId

**Comment by blgrossMS** [comment]
Thanks so much for adding animation. This is a great change!

**Pull request merged**